backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -741,6 +751,14 @@ class SchedulingService implements ISchedulingService {
           `${donor.businessName} Donation Cancellation Confirmation`,
           cancellationEmail(adminMainLine, "Community Fridge KW"),
         );
+        if (volunteer) {
+          this.emailService.sendEmail(
+            volunteer.email,
+            `Cancellation Notice: Food Rescue Shift for ${startDayString} at ${startTimeString}`,
+            cancellationEmail(volunteerMainLine, volunteer.firstName, true),
+          );
+        }
```

Comment:
removed this - I changed it so that in the `schedulingCancellationEmail` function, it only covers sending emails to donor and admin, I put volunteers in a separate function `sendVolunteerFoodRescueEmail`